packages/devtools/lib/src/ui/elements.dart

```diff
@@ -410,6 +414,28 @@ class CloseButton extends CoreElement {
   CloseButton() : super('div', classes: 'close-button');
 }
 
+class TextField extends CoreElement {
+  TextField({String c}) : super('input', classes: c) {
+    setAttribute('type', 'text');
+  }
+
+  String get value => (this.element as InputElement).value;
```

Comment:
nit: Override the Element getter in this class to avoid having repeated casts to InputElement.

Reply:
Good idea plus implicit case seems nicer too:

```
  @override
  InputElement get element => super.element;

  String get value => element.value;

  set value(String newValue) {
    element.value = newValue;
  }
```
Done.